packages/nx-playwright/src/executors/playwright-executor/executor.ts

```diff
@@ -15,6 +15,12 @@ function getFlags(options: PlaywrightExecutorSchema): string {
   return flagStrings.join(' ');
 }
 
+const runnerCommands = {
```

Comment:
Why do we need this object? Could we just use `options.runner`?

Reply:
Yeah I guess that makes sense, will update.